### PR TITLE
Generate shorter file names, and thus paths, in E2E tests

### PR DESCRIPTION
**Pull request by julio-lopez**
Reduces name lengths by ~ 1/2.

Motivation: address failures observed in e2e tests in Windows.

### CODE REVIEW

File: tests/testenv/cli_test_env.go [hunk 1]
```diff
@@ -426,10 +426,11 @@ func randomName(opt DirectoryTreeOptions) string {
 	maxNameLength := intOrDefault(opt.MaxNameLength, 15)
 	minNameLength := intOrDefault(opt.MinNameLength, 3)
 
-	b := make([]byte, rand.Intn(maxNameLength-minNameLength)+minNameLength)
+	l := rand.Intn(maxNameLength-minNameLength) + minNameLength
+	b := make([]byte, (l+1)/2)
```
Comment (golangcibot) [inline tests/testenv/cli_test_env.go]: mnd: Magic number: 2, in <argument> detected (from `gomnd`)

Comment (julio-lopez) [reply tests/testenv/cli_test_env.go]:
```suggestion
	b := make([]byte, (l+1)/2) // nolint:gomnd
```

File: tests/testenv/cli_test_env.go [hunk 2]
```diff
@@ -426,10 +426,12 @@ func randomName(opt DirectoryTreeOptions) string {
 	maxNameLength := intOrDefault(opt.MaxNameLength, 15)
 	minNameLength := intOrDefault(opt.MinNameLength, 3)
 
-	b := make([]byte, rand.Intn(maxNameLength-minNameLength)+minNameLength)
+	l := rand.Intn(maxNameLength-minNameLength) + minNameLength
```
Comment (redgoat650) [inline tests/testenv/cli_test_env.go]: I think this should be 
```suggestion
	l := rand.Intn(maxNameLength-minNameLength + 1) + minNameLength
```
At least when I think of "MaxNameLength" I'd expect the output possibilities should be inclusive of that value.

File: tests/testenv/cli_test_env.go [hunk 3]
```diff
@@ -426,10 +426,12 @@ func randomName(opt DirectoryTreeOptions) string {
...
+	b := make([]byte, (l+1)/2) // nolint:gomnd
+
 	cryptorand.Read(b) // nolint:errcheck
 
-	return hex.EncodeToString(b)
+	return hex.EncodeToString(b)[:l]
```
[the remark this answers is not in the document]
Comment (jkowalski) [reply tests/testenv/cli_test_env.go]: good catch!

[the remark this answers is not in the document]
Comment (jkowalski) [reply tests/testenv/cli_test_env.go]: btw, it would be cool to use less boring names here, perhaps different classes of characters & unicode corner cases (separate PR of course)

File: tests/testenv/cli_test_env.go (hunk 2 above)
Comment (jkowalski) [reply tests/testenv/cli_test_env.go]: +1 makes intuitive sense

File: tests/testenv/cli_test_env.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (julio-lopez) [reply tests/testenv/cli_test_env.go]: Filed #228 last night

File: tests/testenv/cli_test_env.go [hunk 4]
```diff
@@ -426,10 +426,12 @@ func randomName(opt DirectoryTreeOptions) string {
 	maxNameLength := intOrDefault(opt.MaxNameLength, 15)
 	minNameLength := intOrDefault(opt.MinNameLength, 3)
 
-	b := make([]byte, rand.Intn(maxNameLength-minNameLength)+minNameLength)
+	l := rand.Intn(maxNameLength-minNameLength + 1) + minNameLength
```
Comment (golangcibot) [inline tests/testenv/cli_test_env.go]: File is not `gofmt`-ed with `-s` (from `gofmt`)
```suggestion
	l := rand.Intn(maxNameLength-minNameLength+1) + minNameLength
```
